### ISSUE TITLE
Support use of formulas in property, units, precision, etc.

### ISSUE THREAD

**Issue by szeke** [usc-isi-i2/t2wml#100]
In general, formulas can be used to compute the value of any attribute in a YAML file. For example:
```
# OECD mapping
statementMapping:
  region: 
    - range: D6:K12
      skip_cell:
        - =value[$col, $row] == " .."
  template:
    # The next line should wikify the extracted country
    #item: '=get_item(regex(value[B, 2], "profile: (.*) \d{4}", 1))' 
    item: Q31 # Belgium
    property: =item[B, $row]
    value: =replace(value[$col, $row], "[^\d.-]", "")
    unit: =item[C, $row]
    qualifier:
      - property: P585 #point in time
        value: =value[$col, 4]
        calendar: Q1985727
        precision: year
        time_zone: 0
        format: "%Y"
    reference: 
      - property: P246 # stated in
        value: Q41550 # OECD
      - property: P2006010001 # Datamart dataset id
        value: Q2006050001 # OECD dataset
```

In this example, formulas are used in `property` and `unit`, but in general formulas could be used anywhere.

**Comment by devowit** [comment]
formulas _can_ already be used anywhere. The problem is with the formulas in this example.

is there a wikifier file attached to this example that you have not included that assigns items to the cells you're retrieving here? if there is, please attach it, so i can debug further.

if there is not, then that's the problem... you're retrieving items for B, row and D, row but no item exists for those cells. (yes we need clearer error reporting to user for when this happens)

**Comment by devowit** [comment]
hm, no, i also see a bug in general for using it anywhere, fixing now.

but for future such bugs please attach the wikifier file.

**Comment by devowit** [comment]
I see the wikifier file attached in the other bug report works here. so I was able to test the yaml attached. bug is fixed in commit 5c8cd77
